keras/utils/layer_utils.py

```diff
@@ -5,6 +5,11 @@
 import numpy as np
 
 
+def count_params(weights):
+    """Count the total number of scalars composing the weights"""
```

Comment:
Docstring formatting: one-line summary should end with a period; the dosctring should include a `# Arguments` and `# Returns` section.